lite2/client.go

```diff
@@ -655,13 +643,14 @@ func (c *Client) Witnesses() []provider.Provider {
 // client must be stopped at this point.
 func (c *Client) Cleanup() error {
 	c.logger.Info("Removing all the data")
-	return c.cleanup(0)
+	return c.cleanup(0, 0)
 }
 
-// cleanup deletes all headers & validator sets between +stopHeight+ and latest
-// height included. It also sets trustedHeader (vals) to the latest header
-// (vals) if such exists.
-func (c *Client) cleanup(stopHeight int64) error {
+// cleanup deletes all headers & validator sets between startHeight and stopHeight (inclusive).
+// Using a height of 0 will default to the oldest and newest heights respectively.
+// It also resets latestTrustedHeader to the latest header.
+func (c *Client) cleanup(startHeight, stopHeight int64) error {
```

Comment:
does stopHeight ever change? looks like it's 0 in both cases

Reply:
the `Prune()` func has a different stopHeight